tests/resolvers/Mutation/createEvent.spec.ts

```diff
@@ -1221,27 +1221,19 @@ describe("Check for validation conditions", () => {
         data: {
           organizationId: testOrganization?.id,
           allDay: false,
-          description: "newDescription",
-          endDate: new Date().toUTCString(),
-          endTime: new Date().toUTCString(),
+          description: "Random",
+          endDate: "",
+          endTime: "2024-03-02T06:00:00.000Z",
           isPublic: false,
           isRegisterable: false,
           latitude: 1,
           longitude: 1,
-          location: "newLocation",
+          location: "Random",
           recurring: false,
-          startDate: new Date().toUTCString(),
-          startTime: new Date().toUTCString(),
-          title: "newTitle",
+          startDate: "",
+          startTime: "2024-03-02T08:00:00.000Z",
+          title: "Random",
           recurrance: "DAILY",
-          images: [
-            "image_url_1.jpg",
-            "image_url_2.jpg",
```

Comment:
1. In the event test, you have removed image upload testing which was a valid test. A PR was recently merged to add and test that functionality.
2. Please add a new test for this bug

Reply:
Hi @palisadoes , I re-added the tests for image uploads, please have a look